system/ee/ExpressionEngine/View/addons/index.php

```diff
@@ -1,348 +1,290 @@
-                	</div>
-                </div>
-        			</div>
-    					<div class="filter-bar__item ">
-        				<div class="filter-search-bar__item">
-                	<button type="button" class="has-sub filter-bar__button js-dropdown-toggle button button--default button--small" data-filter-label="show">show				<span class="faded">(25)</span>
-            			</button>
-                	<div class="dropdown">
-            				<div class="dropdown__search">
-                			<div class="search-input">
-                  			<input type="text" name="perpage" value="" placeholder="custom limit" data-threshold="1000" data-threshold-text="Viewing more than 1000 items at a time may result in reduced performance." class="search-input__input input--small">
-                			</div>
-                		</div>
-										<a class="dropdown__link" href="">25 results</a>
-										<a class="dropdown__link" href="">50 results</a>
-										<a class="dropdown__link" href="">75 results</a>
-										<a class="dropdown__link" href="">100 results</a>
-										<a class="dropdown__link" href="">150 results</a>
-										<a class="dropdown__link" href="">All Add-Ons</a>
-      						</div>
-                </div>
-        			</div>
-          	</div>
-          </div>
-        </div>
-      </div>
-      <div class="table-responsive table-responsive--collapsible">
-        <table cellspacing="0">
-  				<thead>
-      			<tr class="app-listing__row app-listing__row--head">
-							<th class="column-sort-header column-sort-header--active">
-								<a href="" class="column-sort column-sort--desc">Name</a>
+	<div class="tbl-ctrls">
+		<div class="panel-heading">
+			<div class="app-notice-wrap"><?= ee('CP/Alert')->getAllInlines() ?></div>
+			<?php if (!empty($updates) && count($updates) == 1) : ?>
+				<div id="alert_banner" class="alert">
+					<div class="alert__icon"><i class="fas fa-info-circle fa-fw"></i></div>
+					<div id="alert" href="" class="alert__close">
+						<a id="alert_close" class="fas fa-times alert__close-icon"></a>
+					</div>
+					<p class="alert__title"> <br>
+					<p class="alert__title"><?php echo lang('update'); ?><br>
+						<?php echo lang('there_is'); ?> <strong><?= count($updates) ?></strong> <?php echo lang('single_update'); ?> <a id="ShowMe"><?php echo lang('show'); ?></a></p>
+				</div>
+			<?php endif; ?>
+
+			<?php if (!empty($updates) && count($updates) > 1) : ?>
+				<div id="alert_banner" class="alert">
+					<div class="alert__icon"><i class="fas fa-info-circle fa-fw"></i></div>
+					<div id="alert" href="" class="alert__close">
+						<a id="alert_close" class="fas fa-times alert__close-icon"></a>
+					</div>
+					<p class="alert__title"><?php echo lang('updates'); ?><br>
+
+						<?php echo lang('there_are'); ?> <strong><?= count($updates) ?></strong> <?php echo lang('multi_updates'); ?> <a id="ShowMe"><?php echo lang('show'); ?></a></p>
```

Comment:
Same mix of `=` and `echo` as I've mentioned before. Not saying it's real problem, but if we can avoid it, let's avoid it.